.mjmlconfig

```diff
@@ -1,5 +1,5 @@
 {
   "packages": [
-   "/Users/iryusa/Workspace/test/lib/MjColumnImageText.js"
+    "/home/meri/git/mjml-component-boilerplate/lib/MjColumnImageText.js"
```

Comment:
it feel very weird to have non-project related path to be committed as source file. The best solution would be to use relative path `./lib/MjColumnImageText.js`, but this doesn't work.

IMHO we should put this file in `.gitignore` cause it makes no sense to commit it.